linux_kinect/Bit data
---
bool time_delay
sensor_msgs/Image image